src/core/build_target.go

```diff
@@ -201,6 +201,8 @@ type BuildTarget struct {
 	OutputDirectories []OutputDirectory `name:"output_dirs"`
 	// RuleMetadata is the metadata attached to this build rule. It can be accessed through the "get_rule_metadata" BIF.
 	RuleMetadata interface{} `name:"config"`
+	// EntryPoints represent named binaries within the rules output that can be targeted via //package:rule|entry_point_name
+	EntryPoints map[string]string
```

Comment:
add the `name:"entry_points"` tag, that will make `plz query print` show it nicely